spectral_cube/cube_utils.py

```diff
@@ -504,7 +504,9 @@ def convert_bunit(bunit):
         try:
             unit = u.Unit(bunit)
         except ValueError:
-            warnings.warn("Could not parse unit {0}".format(bunit),
+            warnings.warn("Could not parse unit {0}. "
+                    "If you know the correct unit, try "
+                    "u.add_enabled_units(u.def_unit(['{0}'], represents=u.correct_unit))".format(bunit),
```

Comment:
Maybe add `<>` to make it syntactically invalid until/unless the user has replaced `correct_unit` with an actual valid unit?
```suggestion
                    "u.add_enabled_units(u.def_unit(['{0}'], represents=u.<correct_unit>))".format(bunit),
```